doc/How-to-build.md

```diff
@@ -1,11 +1,125 @@
-# How to compile
-- Windows 64-bit  
-  - Tools needed: Visual Studio 2019, Cmake, git, Strawberry Perl.
-  - Run `build_release.bat` in `x64 Native Tools Command Prompt for VS 2019`
+# How to Compile
 
-- Mac 64-bit  
-  - Tools needed: Xcode, Cmake, git, gettext
-  - run `build_release_macos.sh`
+## Windows 64-bit
 
-- Ubuntu  
-  - run `BuildLinux.sh -udisr`
\ No newline at end of file
+### Tools Required
+- [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) or Visual Studio 2019  
+- [CMake (version 3.31)](https://cmake.org/) — **⚠️ version 3.31.x is mandatory**
+- [Strawberry Perl](https://strawberryperl.com/)
+- [Git](https://git-scm.com/)
+
+### Instructions
+1. Clone the repository:
+   ```sh
+   git clone https://github.com/SoftFever/OrcaSlicer
+   ```
+2. Open the appropriate command prompt:
+   - For Visual Studio 2019:  
+     Open **x64 Native Tools Command Prompt for VS 2019** and run:
+     ```sh
+     build_release.bat
+     ```
+   - For Visual Studio 2022:  
+     Open **x64 Native Tools Command Prompt for VS 2022** and run:
+     ```sh
+     build_release_vs2022.bat
+     ```
+
+**⚠️ Note 1:** Make sure that CMake version 3.31.x is actually being used. Run `cmake --version` and verify it returns a **3.31.x** version.
+If you see an older version (e.g. **3.29), it's likely due to another copy in your system's PATH (e.g. from Strawberry Perl).
+You can run where cmake to check the active paths and rearrange your System Environment Variables > PATH, ensuring the correct CMake (e.g. C:\Program Files\CMake\bin) appears before others like C:\Strawberry\c\bin.
+
+**⚠️ Note 2:** If the build fails, delete the entire project directory, re-clone the repository, and try again to ensure a clean cache.
```

Comment:
We should remove this IMO
Delete the build and deps/build folder is sufficient.
But anyway delete building folder or project folder is not a recommended way to fix build errors. And shouldn't be a tip/note